packages/messaging/package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.2.3",
   "description": "",
   "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
   "scripts": {
-    "dev": "gulp dev",
+    "build": "rollup -c",
+    "dev": "rollup -c -w",
     "test": "run-p test:karma type-check lint",
     "test:karma": "karma start --single-run",
     "test:debug": "karma start --browsers=Chrome --auto-watch",
-    "prepare": "gulp build",
+    "prepare": "npm run build",
```

Comment:
Should these be "yarn build"?

Reply:
If we were leveraging `yarn` as a `devDependency` here, yes. But since we can't assume yarn exists, sticking w/ NPM is functionally equivalent, and is safer.